stock_picking_back2draft/README.rst

```diff
@@ -0,0 +1,54 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+======================
+Pickings back to draft
+======================
+
+This module allows to bring cancelled pickings back to draft
+
+Usage
+=====
+
+Just click on 'back to draft' button
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/154/8.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/stock-logistics-workflow/issues>`_. In case of trouble, please
```

Comment:
I followed the template at https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
Is it missing some part?